README.md

```diff
@@ -23,5 +23,5 @@ If you are interested in contributing, see [CONTRIBUTING.md](./CONTRIBUTING.md),
 [DEVELOPMENT.md](./DEVELOPMENT.md) and
 [Knative WORKING-GROUPS.md](https://github.com/knative/docs/blob/master/community/WORKING-GROUPS.md#events).
 
-The planned project releases are covered in this
+Please join [knative-users](https://groups.google.com/forum/#!forum/knative-users) to view planned project releases in
```

Comment:
Maybe add a link to the https://github.com/knative/docs/blob/master/community/WORKING-GROUPS.md page as well which documents both groups for view and comment.